src/components/Calculator.js

```diff
@@ -1,28 +1,27 @@
-import React from 'react';
```

Comment:
- [ ] Please do not include `.babelrc`, `eslint.json`, and `.stylelintrc.json` files in `.gitignore`, as these files after they're installed extend the initial config from create-react-app and become important to compile the project.

![image](https://user-images.githubusercontent.com/79114233/144712579-a3625e4e-88eb-4066-88e0-d647ba2f9684.png)

Reply:
Fixed!